.gitlab-ci.yml

```diff
@@ -394,66 +373,52 @@ Pub:Internal::WP40:HL:
     - Test_N_Build:internal
     - Node:Internal
 
-# Job to deploy review deployments to development environment in Hinxton
-Review:HX:
-  extends: .deploy-review
+# Job to deploy to live environment in the new kubernetes cluster (wp40)
+Dev:wp51:HL:
```

Comment:
The comment on line `376` contradicts the name of the job on line `377` (different cluster names; comment says live but the job name says dev)